File: TP1 java/java fede/Ejercicio 3/Serie.java
public class Serie {
    
    private String titulo;
    private String urlJpg;
    private String estreno;

    public Serie(String titulo, String urlJpg, String estreno){
        this.urlJpg = urlJpg;
        this.titulo = titulo;
        this.estreno = estreno;
    }

    public void imprimirSer(){
        System.out.println(this.urlJpg + "   " + this.titulo + "  " + this.estreno);
    }
}

File: TP1 python/python fede/ejercicio 3/Serie.py
class Serie:
    
    def __init__(self, urlJpg, titulo, estreno):
        self.urlJpg = urlJpg
        self.titulo = titulo
        self.estreno = estreno
        
    def imprimirSerie(self):
        print("{}       {}       {}".format(self.urlJpg, self.titulo, self.estreno))